### PR TITLE
WIP add ssb-peer-invites

### PR DESCRIPTION
**Pull request by mixmix**
This is to address https://github.com/ssbc/patchbay/issues/341

### CODE REVIEW

File: app/page/network/connections.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      title: 'local peers',
+      body: h('LocalPeers', [
+        computed(state.localPeers, peers => {
+          if (!peers.length) return h('p', 'No local peers (on same wifi/ LAN)')
+
+          return peers.map(peer => avatar(peer))
+        }),
+        h('p', [
+          h('i.fa.fa-info-circle'),
+          'these are people on the same WiFi/ LAN as you right now. You might not know some of them yet, but you can click through to find out more about them and follow them if you like.'
```
Comment (arj03) [inline app/page/network/connections.js]: WiFi/ LAN ->  WiFi/LAN

File: app/page/network/connections.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      title: 'remote peers',
+      body: h('RemotePeers', [
+        computed(state.remotePeers, peers => {
+          if (!peers.length) return h('p', 'No remote peers connected')
+
+          return peers.map(peer => avatar(peer))
+        }),
+        h('p', [
+          h('i.fa.fa-info-circle'),
+          'these are people on the same WiFi/ LAN as you right now. You might not know some of them yet, but you can click through to find out more about them and follow them if you like.'
```
Comment (arj03) [inline app/page/network/connections.js]: This should be a different text (remote peers), right?

File: app/page/network/invite-peer.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+    onceTrue(connection, sbot => {
+      var interval = setInterval(
+        () => {
+          const dt = Date.now() - start
+          const mins = Math.floor(dt / MINUTE)
+          const secs = Math.floor((dt - mins * MINUTE) / SECOND)
+          state.create.time.set(`${mins} mins, ${secs} seconds`)
+        },
+        1e3
```
Comment (arj03) [inline app/page/network/invite-peer.js]: SECOND

File: app/page/network/replication-in.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
+const { h, Value, Dict, onceTrue, computed, watch, watchAll, throttle } = require('mutant')
```
Comment (arj03) [inline app/page/network/replication-in.js]: wooah, nice! This file is really sweet. Gonna be testing this.

File: app/page/network/replication-in.js [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+    const _data = Object.keys(data)
+      .sort((a, b) => a < b ? -1 : +1)
+      .map(ts => {
+        return {
+          t: Number(ts), // NOTE - might need to offset by a half-step ?
+          y: data[ts]
+        }
+      })
+
+    // update chard data
```
Comment (arj03) [inline app/page/network/replication-in.js]: chart

**Comment by arj03** [comment]
Had 10 minutes, so did a quick code skim. Really looking forward to testing this. A lot of the network stuff would be much easier to debug with tools like these I think, so great work!

**Comment by mixmix** [comment]
Same tools, nicer layour @arj03 !

File: app/page/network/connections.js (hunk 2 above)
Comment (mixmix) [reply app/page/network/connections.js]: nice catch!

File: app/page/network/replication-in.js (hunk 4 above)
Comment (mixmix) [reply app/page/network/replication-in.js]: you've already seen this in the network page right ... it's been in patchbay for a while!

or have you not seen `/network` !

File: app/page/network/replication-in.js (hunk 5 above)
Comment (mixmix) [reply app/page/network/replication-in.js]: I stand by my typo (/ can't be bothered opening that file to fix!)

**Comment by mixmix** [comment]
Here's some screenshot of the final work! (not much to see)

Selection_402

**Pull request merged**